bridge_adaptivity/api/tests/test_api.py

```diff
@@ -61,7 +61,6 @@ def test_create_collection(self, mock_apply_async):
         name = 'test_collection'
         data = {
             'name': name,
-            'slug': f'{name}1',
```

Comment:
Because I added a slug field to CollectionOrder model. This model has two filed: collection and group. I used CollectionOrder's slug to find a collection in a group with a specific engine and policy. So a slug field in Group and Collection models are excess because of these fields unused.